src/runtime/app/composables/session.ts

```diff
@@ -38,12 +49,48 @@ export function useUserSession(): UserSessionComposable {
     }
   }
 
+  const isInPopup = useCookie('temp-nuxt-auth-utils-popup')
+
+  const openInPopup = (provider, route) => {
```

Comment:
I would rather have `onpenInPopup('/auth/github', { width: 690, height: 600 })` with default values for both width & height so we could have `openInPopup('/auth/github')`

Reply:
As websites can evolve, I don't want to myself maintain the list of with & height for each provider, also, this would make the app bundle size bigger to store all of them when in reality you may use only one of two

Reply:
You're right. We can just provide the best width & height for each provider in the readme.md